src/odb/src/db/dbModuleBTermItr.cpp

```diff
@@ -80,7 +82,7 @@ uint dbModuleBTermItr::begin(dbObject* parent)
 {
   // User Code Begin begin
   _dbModule* _module = (_dbModule*) parent;
-  _dbModBTerm* _modbterm = _module->_modbterms;
+  return _module->_modbterms;
```

Comment:
Actually seems like a bug fix? I'm surprised the compiler didn't complain about not returning a value?

Reply:
I see this file isn't in the CMakeLists.txt so it isn't even compiled.  How did you happen upon this error?

Reply:
@andyfox-rushc did you forget to the commit the json for this file?

Reply:
@andyfox-rushc I see dbModInstITermItr, dbModInstModITermItr, dbModuleBTermItr, dbModuleModInstITermItr all have this issue.

Reply:
We are using our own bazel scripts instead of cmake to build. In our setup we were building everything in this directory, but this seemed better to fix upstream than to exclude.

Reply:
These files need to be removed as they are redundant. The came from a prior mis-naming in the schema. All the new dbModule iterators are prefixed by dbModuleMod*. So this is my error. New pull request to be issued removing these unused files.